Test failure: pause_at_block_test

https://github.com/AntelopeIO/spring/actions/runs/15824426576/job/44601955281

```
Traceback (most recent call last):
  File "/__w/spring/spring/build/tests/pause_at_block_test.py", line 70, in <module>
    assert headNode.getHeadBlockNum() == blockNum, "Head Node_03 did not pause at block"
AssertionError: Head Node_03 did not pause at block
```

--- a/tests/pause_at_block_test.py
+++ b/tests/pause_at_block_test.py
@@ -64,6 +64,9 @@ try:
 
     assert prodNode.waitForLibToAdvance(), "LIB did not advance with paused nodes"
     assert prodNode2.waitForBlock(blockNum), f"Block {blockNum} did not arrive after pausing"
+    assert headNode.waitForBlock(blockNum), f"Block {blockNum} did not arrive after pausing"
+    assert specNode.waitForBlock(blockNum), f"Block {blockNum} did not arrive after pausing"
+    assert irrvNode.waitForBlock(blockNum), f"Block {blockNum} did not arrive after pausing"
 
     Utils.Print(f"Verify paused at block {blockNum}")
     assert prodNode2.getHeadBlockNum() == blockNum, "Prod Node_01 did not pause at block"

